src/Form/Control/Upload.php

```diff
@@ -10,7 +10,7 @@ function init(): void {
         parent::init();
 
         $this->onUpload(function ($files) {
-            $this->uploaded($files); 
+            $this->uploaded($file); 
```

Comment:
must be typo

Reply:
Was fixed above already. we should go for $files as in the original Upload class. This should be capable of handling multiple file uploads.